fastanvil/src/complete/section.rs

```diff
@@ -0,0 +1,41 @@
+use crate::{java, Block};
+
+pub struct Section {
+    block_pallet: Vec<Block>,
+
+    //could be [] because of fix size
+    //This will increase in x, then z, then y.
+    blocks: Option<Vec<u16>>,
+}
+
+impl Section {
+    pub fn from_current_section(current_tower: &java::Section) -> Self {
```

Comment:
Feels like it should be possible for this to take by value as well.

Reply:
i dont think it is possible with the current `java:SectionTower` api.
simply becouse there is no way to acces the sections directly. only though a `&[]`

Reply:
I think it would be fine to add a `pub(crate) take_sections(self) -> Vec<S>` method to `SectionTower`. That way only `fastanvil` has access.